[doc] Update tutorial-todo-gql-field-collection.md (#2315)
ent/ent: doc/md/tutorial-todo-gql-field-collection.md

	}
}
```
:::info

> Bind() annotation now enabled by default, and use Unbind() to opt-out

Reviewer: ```suggestion
The `Bind()` annotation is now enabled by default, use `Unbind()` to opt-out.
```
